### PR TITLE
Move autobooting steps to autobooting section(round 2) + Minor changes

### PR DESCRIPTION
**Pull request by VannyBuns**
Also added info about how to merge on Mac.

### CODE REVIEW

File: docs/user-guide/aroma/autoboot.md [hunk 1]
```diff
@@ -15,6 +16,22 @@ Currently, every time you want to launch Aroma you will have to launch the Healt
 1. You will be asked if you want to switch the boot title. Press A to select `Switch to PayloadLoader`.
 1. When the process has finished, press A to shutdown the console.
 1. The PayloadLoader will now be launched automatically on every boot.
-    - To change the default boot title hold Start (+) on the Gamepad while booting the console.
+
+
+Now, we are going to make the Aroma environment start automatically when your console autoboots the Health and Safety Information app and select Wii U Menu as the default title.
```
Comment (Lazr1026) [inline docs/user-guide/aroma/autoboot.md:21]: Should move this line under the header, and change it to
```suggestion
Now, we are going to make the Aroma environment start automatically when your console loads the Health and Safety Information app and select Wii U Menu as the default boot option.
```

File: docs/user-guide/aroma/autoboot.md [hunk 2]
```diff
@@ -15,6 +16,22 @@ Currently, every time you want to launch Aroma you will have to launch the Healt
...
+1. Launch the EnvironmentLoader.
+    - If you are autobooting the PayloadLoader, simply turn on your Wii U.
+    - If you skipped the autobooting steps, launch the Health and Safety Information app.
+1. Using the D-Pad, navigate to `aroma` and press Y to set this to your default environment, then press A to launch into Aroma.
+    - You might get a red warning screen telling you that updates aren't blocked properly. Press A to continue anyway. We're going to block updates in the "Blocking Updates" section below.
+    - To open the Environment Loader in the future, you have to hold X while your Wii U is booting or loading the Health and Safety Information app.
+
+1. On the Aroma Boot Selector, the `Wii U Menu` should already be selected, press Y to set this to your default autobooting option, then press A to launch into the Wii U Menu.
+   
+1. Aroma will now be launched each time you boot(or boot Health and Safety) and then straight into the Wii U Menu.
```
Comment (Lazr1026) [inline docs/user-guide/aroma/autoboot.md:34]:
```suggestion
1. Aroma will now be launched each time you boot the console (or load Health and Safety) and then straight into the Wii U Menu.
```

**Review by Lazr1026 — CHANGES_REQUESTED** [review]

File: docs/user-guide/aroma/autoboot.md [hunk 3]
```diff
@@ -15,6 +16,22 @@ Currently, every time you want to launch Aroma you will have to launch the Healt
 1. You will be asked if you want to switch the boot title. Press A to select `Switch to PayloadLoader`.
 1. When the process has finished, press A to shutdown the console.
 1. The PayloadLoader will now be launched automatically on every boot.
-    - To change the default boot title hold Start (+) on the Gamepad while booting the console.
+
+
+Now, we are going to make the Aroma environment start automatically when your console loads the Health and Safety Information app and select Wii U Menu as the default boot option.
```
Comment (Lazr1026) [inline docs/user-guide/aroma/autoboot.md:21]: The only issue left with the PR is this still isn't under the title tag.

Comment (VannyBuns) [reply docs/user-guide/aroma/autoboot.md:21]: For some reason the file on my PC...reset? Tf github

Comment (VannyBuns) [reply docs/user-guide/aroma/autoboot.md]: Okay done

Comment (Lazr1026) [reply docs/user-guide/aroma/autoboot.md]: such a github moment

**Pull request merged**